src/main/java/calculator/view/InputView.java

```diff
@@ -0,0 +1,12 @@
+package calculator.view;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static Scanner sc = new Scanner(System.in);
+
+    public static String inputExpression() {
+        System.out.println("계산식을 입력해주세요.");
+        return (sc.nextLine());
```

Comment:
크게 중요한 부분은 아니지만 불필요한 괄호가 들어가있네요!

```suggestion
        return sc.nextLine();
```